microsoft/testsuites/vm_extensions/linux_patch_extension.py

```diff
@@ -14,17 +14,57 @@
     simple_requirement,
 )
 from lisa.base_tools.service import Service
-from lisa.operating_system import BSD
+from lisa.operating_system import BSD, SLES, CBLMariner, Debian, Redhat
 from lisa.sut_orchestrator import AZURE
 from lisa.sut_orchestrator.azure.common import (
     get_compute_client,
     get_node_context,
     wait_operation,
 )
 from lisa.sut_orchestrator.azure.platform_ import AzurePlatform
+from lisa.util import SkippedException, UnsupportedDistroException
+
+
+def _is_os_unsupported(self, node: Node) -> None:
+    """
+    Below specific Linux distro versions that are
+    not supported by linux patch extension.
+    debian debian-11 11 0.20230124.1270
+    redhat rhel-arm64 8_6-arm64 8.6.2022060901
+    debian debian-11 11-backports 0.20221219.1234
+    microsoftcblmariner cbl-mariner cbl-mariner-2-arm64 2.20230126.01
+    suse sles-15-sp5 gen2 2024.02.07
```

Comment:
no need include the details, list unsupported ones is good enough

Reply:
logic changed